docs/Rakefile

```diff
@@ -78,7 +78,7 @@ task :adoc_tar => [:init, :adoc_variables, :markdown_to_asciidoc] do
       file_list.each { |f|
         tar.add_file(f, 0444) { |io|
           if File.extname(f) == ".adoc" then
-            adoc_content = File.read(f)
+            adoc_content = File.read(f, encoding: "ISO8859-1:utf-8")
```

Comment:
Seems like reading from `.adoc` file, we are getting encoding as `US-ASCII` which was creating the problem.